ocs_ci/utility/performance_dashboard.py

```diff
@@ -38,18 +40,27 @@ def initialize_data():
     Returns:
         dict: A dictionary contains the data to push to the dashboard
     """
-    worker_type = get_typed_nodes(num_of_nodes=1)[0].data['metadata'][
-        'labels'
-    ]['beta.kubernetes.io/instance-type']
+
+    # worker type is relevant only for cloud instances.
+    log.info('Initializing the dashboard data')
+    worker_lbl = get_typed_nodes(num_of_nodes=1)[0].data['metadata']['labels']
+    if 'beta.kubernetes.io/instance-type' in worker_lbl:
+        worker_type = worker_lbl['beta.kubernetes.io/instance-type']
+    else:
+        # TODO: Maybe for None cloud we can add the Arch ?
```

Comment:
+1 
You can check cloud/non-cloud using constants.CLOUD_PLATFORMS

Reply:
Changed it.